operator/gitops/argocd/pipeline-service/tekton-chains/kustomization.yaml

```diff
@@ -39,3 +39,12 @@ patches:
         name: signing-secrets
         namespace: tekton-chains
       $patch: delete
+  # Use downstream image
+  - target:
+      kind: Deployment
+      name: tekton-chains-controller
+      namespace: tekton-chains
+    patch: |-
+     - op: replace
+       path: /spec/template/spec/containers/0/image
+       value: registry.redhat.io/openshift-pipelines/pipelines-chains-controller-rhel8@sha256:f4ad9be541f927d0b0348d4515639f490657e699ca2b0fdfb8552f144793beca
```

Comment:
I'll add what I mentioned in planning @Roming22 for completeness.

Per https://catalog.redhat.com/software/containers/openshift-pipelines/pipelines-chains-controller-rhel8/61f9058895bac8d0de150838 how about trying `v1.10.0-4` instead of a specific sha

The way those tags work in registry.redhat.io, v1.10.0-4 is not a rolling tag, but will only ever point to the sha it is initially seeded with.

The makes it clear to humans when looking at the deployment which version we are using here.

Reply:
Fixed.